### PR TITLE
JBIDE-22575 - Read docker registry from service-manager

### PR DESCRIPTION
**Pull request by robstryker**
Tested with service manager 1.0.1,  1.0.2,  and 1.1.0.

### CODE REVIEW

**Comment by alexeykazakov** [comment]
Any tests?

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java [hunk 1]
```diff
@@ -93,7 +97,9 @@ public static ServiceManagerEnvironment loadServiceManagerEnvironment(IServer se
 	 */
 	public static ServiceManagerEnvironment loadServiceManagerEnvironment(IServer server) {
 		
-		String[] args = new String[]{CDKConstants.VAGRANT_CMD_SERVICE_MANAGER, CDKConstants.VAGRANT_CMD_SERVICE_MANAGER_ARG_ENV, "docker"};
```
Comment (alexeykazakov) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java]: So you removed "docker" from the argument? But what about service manager 1.0.1 (CDK 2.0.0) support?

Comment (fbricon) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java]: @robstryker I'm pretty sure you told me old versions of the service-manager plugin wouldn't support just `vagrant service-manager env`

**Comment by alexeykazakov** [comment]
OK. If you are sure it also works fine with CDK 2.0. (service manager 1.0.1) then +1 for merging to 4.4.0.x.

**Comment by fbricon** [comment]
+1

**Pull request merged**